docs/plugins/drupal-plugin.md

```diff
@@ -14,6 +14,21 @@ See the <a href="#drupal-plugin-examples" title="Probo Drupal Plugin Examples">P
 ## Directory Configuration
 
 {% option_list %}
+{% option %}
+### `configSyncDirectory` {string}
+Specify a the location of the configuration synchronization directory for Drupal 8 sites. Defaults to `sites/default/files/config_UNIQUE_ID/sync`.
```

Comment:
@jkaeser Should we mention where the path you set is relative to? I assume it's the root of your repository (which we know to be in /src). Also, is the UNIQUE_ID part of the default path something we expose as a variable?

Reply:
How is that?